pages/pipelines/block_step.md.erb

```diff
@@ -42,6 +42,14 @@ _Optional attributes:_
       A list of input fields required to be filled out before unblocking the step.<br>
       Available input field types: <code>text</code>, <code>select</code>
     </td>
+  </tr>
+    <tr>
+    <td><code>blocked_state</code></td>
+    <td>
+      The state that the build should be set to when the build is blocked by this block step.<br>
```

Comment:
```suggestion
      The state that the build should be set to when the build is blocked by this block step. The default behaves the same as `passed` in the Buildkite UI and other integrations.<br>
```

Reply:
Thanks for pushing this again @blaknite hat do you think about something along the lines of 

> If you don't set this to something other than the default (`nil`), the build will show as passed in the Buildkite UI and other integrations.

Reply:
Yeah love it!